Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/App.xaml.cs

```diff
@@ -48,6 +55,33 @@ public App()
             MainPage = new NavigationPage(new MainDemoPage());
         }
 
+        private string GetOneCollectorTokenString()
+        {
+            return $"androidTarget={OneCollectorTokens[XamarinDevice.Android]};iosTarget={OneCollectorTokens[XamarinDevice.iOS]}";
+        }
+
+        private string GetAppCenterTokenString()
+        {
+            var appSecrets = GetAppSecretDictionary();
+            return $"uwp={appSecrets[XamarinDevice.UWP]};android={appSecrets[XamarinDevice.Android]};ios={appSecrets[XamarinDevice.iOS]}";
+        }
+
+        private string GetTokensString()
+        {
+            var persistedStartType = StartTypeUtils.GetPersistedStartType();
+            switch (persistedStartType)
+            {
+
+                case StartType.OneCollector:
+                    return GetOneCollectorTokenString();
+                case StartType.Both:
+                    return $"{GetAppCenterTokenString()};{GetOneCollectorTokenString()}";
+                case StartType.AppCenter:
```

Comment:
I guess `case StartType.AppCenter` it can be remove

Reply:
```suggestion
```